index.js

```diff
@@ -3,6 +3,23 @@ const fs = require("fs");
 const webpack = require("./webpack/lib/webpack");
 const MemoryFS = require("memory-fs");
 
+function resolveModule (context, request, callback) {
+  const resolveOptions = {
+    basedir: context,
+    preserveSymlinks: true,
+    extensions: ['.js', '.json']
```

Comment:
I think we need to add all file extensions webpack uses

Reply:
https://webpack.js.org/configuration/resolve/#resolve-extensions

```
extensions: ['.wasm', '.mjs', '.js', '.json']
```

Reply:
Maybe I'm wrong, but I thought the idea of this package was to mimic the behavior of Node, which supports only .js, .json and .node. After all, the fact that ncc uses webpack is an implementation detail, it's not exposed to the end user and they should not expect any webpack-specific features.

Btw, ".node" probably also needs some special treatment, even though it's used quite rarely.

Reply:
Then I guess we have to force `.js` `.json` for webpack's resolve too 👍

Reply:
I've updated the PR 👍